### PR TITLE
Replace hydra with samvera

### LINKED ISSUE #83: change name of Hydra to Samvera throughout docs

### PR DESCRIPTION
**Pull request by mjgiarlo**
Done quickly, needs testing and review!

Fixes #83

### CODE REVIEW

**Comment by laritakr** [comment]
Looks ok to me. I found a couple problems during testing, but they also exist on the live site, so I'm creating separate issues for them.

**Comment by mjgiarlo** [comment]
@laritakr OK, I removed the WIP. Thank you! Does this need more review or more work?

**Comment by laritakr** [comment]
The only thing I might suggest is fixing the Formalities page link. It wasn't a link, but as I was documenting that in the new issue, I found that it's not a correct URL either.  See issue https://github.com/samvera/samvera.github.io/issues/115

**Comment by mjgiarlo** [comment]
@laritakr Done.

File: pages/samvera/developer_community/communication.md [hunk 1]
```diff
@@ -54,9 +54,9 @@ Slack has become the primary mode of synchronous communication in the Hydra comm
 
 ## Training
 
-* [HydraCamp](https://curationexperts.com/our-services/hydra-camp/): for many partners, one of the
-  first steps to joining the Hydra community was, and remains, sending a team of software developers
-  to HydraCamp to learn the common practices of our community. We generally hold 3-4 HydraCamps per
+* [SamveraCamp](https://curationexperts.com/our-services/samvera-camp/): for many partners, one of the
```
Comment (jcoyne) [inline pages/samvera/developer_community/communication.md]: I don't think this is a valid change

Comment (mjgiarlo) [inline pages/samvera/developer_community/communication.md]: Changed back.

**Pull request merged**